drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/models/ScenarioGridModel.java

```diff
@@ -243,7 +244,11 @@ public void duplicateColumnValues(int originalColumnIndex, int newColumnIndex) {
      */
     public void insertColumnGridOnly(final int index, final GridColumn<?> column) {
         checkSimulation();
-        super.insertColumn(index, column);
+
+        column.setIndex(columns.size());
+        columns.add(index, column);
+
+        selectionsManager.onInsertColumn(index);
```

Comment:
This method (and in particular the last 3 lines) could be put in its superclass.